### PR TITLE
Create example-test-scenarios.md

### LINKED ISSUE #1210: Documentation for bulk upload fields
After questions from one of the 1A states, try to provide better in-project documentation of which participants to include.

- [x] How to use the fields
- [ ] Example use cases and test cases

We want the project documentation to reflect the settled definition of active participants:
> An active participant is an individual who is certified to receive SNAP benefits for the current monthly benefit cycle on the day when the bulk upload file is being generated.

### PR DESCRIPTION
**Pull request by cjhskippy**
Provides examples that states can use to ensure that bulk uploads contain the right content after each type of case action.

## What’s changing?

New documentation added to the bulk uploads / ETL

## Why?

Provides more clarity for people implementing and testing bulk uploads, to prevent individuals from being included in bulk uploads who should not be, and to ensure that participants are included in uploads for the full duration that they are expected to be.

Resolves #1210

## This PR has:

- [x] **Developer documentation** (_for new build/test/API changes or complex portions of the system_)

### CODE REVIEW

File: etl/docs/example-test-scenarios.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+Only active participants can be included in bulk uploads.  An active participant is an individual who is certified to receive SNAP benefits for the current monthly benefit cycle on the day when the bulk upload file is being generated. 
+
+When filtering participant records for upload, you will need to consider these factors: 
+
+- Has the individual been certified for benefits? 
+- Has the household already received benefits for the individual in the current month’s benefit cycle? 
+- Will the household receive benefits for the individual in the current month’s benefit cycle? 
+
+## Examples 
+The following examples can be used to generate test cases to help you ensure that your state is uploading the correct participant records.
```
Comment (kingcomma) [inline etl/docs/example-test-scenarios.md:12]: *Question:* Is it worth clarifying that these examples assume a benefit month begins on the 1st and ends on the final day of the month?

File: etl/docs/example-test-scenarios.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+_Daily bulk upload contents:_
+
+- October 1 - November 30: Only the previously certified individuals are included 
+- December 1+: The newly certified individual of the household is now included
+
+
+### Removing a household member 
+
+#### Household member removal example: 
+A member of a household moves out of the house.  The household submits documentation to remove a household member on October 15.  The household received benefits for this individual for the month of October.
```
Comment (kingcomma) [inline etl/docs/example-test-scenarios.md:52]:
```suggestion
A member of a household moves out of the house.  The household submits documentation to remove a household member on October 15.  The household received benefits for this individual for the entire month of October. 
```

*Question:* Would it me more clear to use this suggested text? Trying to differentiate from "The household received benefits for this individual _at some point_ in the month of October." Or perhaps the current language better fits the expected parlance?

File: etl/docs/example-test-scenarios.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+#### Case closure example 1: 
+A household closed its case on October 15.  Prior to closing their case, the household received benefits for the month of October. 
+
+_Daily bulk upload contents:_
+- October 1 - October 31: Each certified member of this household must be included 
+- November 1+: Each member of this household is excluded. 
+ 
+#### Case closure example 2: 
+A household closes its case on October 20, with the last issuance taking place in November.
```
Comment (kingcomma) [inline etl/docs/example-test-scenarios.md:69]: Ditto again.

**Review by kingcomma — COMMENTED** [review]
Nice! I have just a couple questions on the language around months — there are a few places where I'm sure if I should read "month of X" as meaning "entire month of X" or "at any point in the month of X."

You definitely understand the parlance better than I do at this point, so feel free to reject them all 😄

**Review by kingcomma — APPROVED** [review]
👍

**Review by adabbraccio-ventera — APPROVED** [review]
Updated review - no changes needed

**Pull request merged**